manifests/logical_volume.pp

```diff
@@ -28,6 +28,7 @@
   $no_sync                           = undef,
   $region_size                       = undef,
   $alloc                             = undef,
+  Boolean $yes                       = true,
```

Comment:
I'm wondering if this would be a breaking change because it changes the default behaviour. maybe it makes sense to default to false?

Reply:
I think it makes sense to merge this with false as default value and then create a new PR that changes it to true. but we will keep that on hold for the next major release.